### ISSUE TITLE
cannot build

### ISSUE THREAD

**Issue by MrOutput** [shuLhan/rescached-legacy#7]
could not build on gentoo

```
[M] mkdir lib/build
[M] mkdir build
[C] build/main.oo
[C] build/common.oo
[C] build/Rescached.oo
[C] build/ResQueue.oo
[C] build/NameCache.oo
[C] build/NCR.oo
[C] build/NCR_Tree.oo
[C] lib/build/libvos.oo
[C] lib/build/Object.oo
[C] lib/build/Buffer.oo
[C] lib/build/File.oo
[C] lib/build/DSVRecordMD.oo
[C] lib/build/DSVRecord.oo
[C] lib/build/DSVWriter.oo
[C] lib/build/DSVReader.oo
[C] lib/build/Locker.oo
[C] lib/build/SockAddr.oo
[C] lib/build/Socket.oo
[C] lib/build/SockServer.oo
[C] lib/build/Resolver.oo
[C] lib/build/DNS_rr.oo
[C] lib/build/DNSQuery.oo
[C] lib/build/Dlogger.oo
lib/Dlogger.cc: In member function ‘void vos::Dlogger::_w(int, const char*)’:
lib/Dlogger.cc:120:32: error: ignoring return value of ‘ssize_t write(int, const void*, size_t)’, declared with attribute warn_unused_result [-Werror=unused-result]
   ::write(fd, _tmp._v, _tmp._i);
                                ^
cc1plus: all warnings being treated as errors
lib/Makefile:176: recipe for target 'lib/build/Dlogger.oo' failed
make: *** [lib/build/Dlogger.oo] Error 1

```

**Comment by shuLhan** [comment]
Thank you for reporting the issue.

Can you provide your CFLAGS and LDFLAGS?

**Comment by MrOutput** [comment]
cflags are `-march=silvermont -O2 -pipe` ld flags have not been changed

**Comment by shuLhan** [comment]
I can not reproduce the issue on my ArchLinux system using GCC 6.2.1; but I have try to fix them on recent commit (fad222f9).

**Comment by MrOutput** [comment]
thank you, that fixed the problem!
